src/index.ts

```diff
@@ -11,3 +11,4 @@ export * from './components';
 export * from './hooks';
 export * from './screens';
 export * from './navigators';
+export * from './common/SecureStore';
```

Comment:
I think you can back this change out.  `SecureStore` is already exported from `./common`